src/main/java/io/beanmapper/core/BeanMatch.java

```diff
@@ -106,6 +111,7 @@ private void checkForMandatoryUnmatchedNodes(String side, Class<?> containingCla
         for (Map.Entry<String, BeanProperty> entry : nodes.entrySet()) {
             BeanProperty currentField = entry.getValue();
             if (currentField.isUnmatched()) {
+                log.error("{} {} has no match for property {}", side, containingClass.isAnonymousClass(), entry.getKey());
```

Comment:
Is het echt een error als het veld niet is gematched?

Reply:
Ja, in een dergelijk geval wordt de mapping afgebroken.